### PR TITLE
fix(plugin-navtree): Use dropdown menu again for tree item actions

### LINKED ISSUE #5578: [navtree] show actions menu on hover on desktop
the current right-click affordance is a bit undiscoverable for web users

on desktop, let's experiment with the actions menu showing on hover?

### PR DESCRIPTION
**Pull request by thure**
Resolves #5578.

This PR switches NavTree items back to using a dropdown menu for actions as before.

https://github.com/dxos/dxos/assets/855039/a89bea5a-9716-4f3d-85ac-153f952d24c4

https://github.com/dxos/dxos/assets/855039/052b38e2-fa8c-4879-b31d-1e2541d0ccc1

### CODE REVIEW

**Review by wittjosiah — APPROVED** [review]

File: packages/apps/composer-app/src/playwright/app-manager.ts
[the remark this answers is not in the document]
Comment (thure) [reply packages/apps/composer-app/src/playwright/app-manager.ts:1]: I just hope I didn’t miss something! Will give this CI trick a go and see.

[the remark this answers is not in the document]
Comment (thure) [reply packages/apps/composer-app/src/playwright/app-manager.ts:1]: e2e is failing, but the specific tests that are failing don’t rely on treeitem actions (or if they do, they rely on functions that other passing tests also use).

[the remark this answers is not in the document]
Comment (wittjosiah) [reply packages/apps/composer-app/src/playwright/app-manager.ts:1]: I can take a look tomorrow and see if I can help explain what's going on. I've spend a bunch of time looking at these recently so I've gotten better at tracking down some of the quirks that come up.

[the remark this answers is not in the document]
Comment (wittjosiah) [reply packages/apps/composer-app/src/playwright/app-manager.ts:1]: I realized that your branch is just missing the fix I pushed to the RC prior to the release. I've updated from main and running again, let's see if it passes now.

**Pull request merged**